src/vma/dev/cq_mgr.cpp

```diff
@@ -526,8 +526,7 @@ mem_buf_desc_t* cq_mgr::process_cq_element_rx(vma_ibv_wc* p_wce)
 	// Get related mem_buf_desc pointer from the wr_id
 	mem_buf_desc_t* p_mem_buf_desc = (mem_buf_desc_t*)(uintptr_t)p_wce->wr_id;
 
-	bool bad_wce = (p_wce->status != IBV_WC_SUCCESS) ||
-			(m_b_is_rx_csum_on && !vma_wc_rx_csum_ok(*p_wce));
+	bool bad_wce = p_wce->status != IBV_WC_SUCCESS;
```

Comment:
1. It seems that before this PR if CSUM was wrong - we dropped the packet. 
```
bool bad_wce = (p_wce->status != IBV_WC_SUCCESS) ||
m_b_is_rx_csum_on && !vma_wc_rx_csum_ok(*p_wce)
```
(assuming m_b_is_rx_csum_on is true)
Starting from this PR if safe_mce_sys().rx_sw_csum if false - we stop dropping packets with bad CSUM.
If it is correct - need to fix in order to be backward compatible.
2. Please let know what does the HW CSUM check include: 
```
vma_wc_rx_csum_ok(*p_wce)
```
IP only or both IP and TCP/UDP?